app/cisc/lib.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"io/ioutil"
+	"os"
+	"sort"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/network"
+	"github.com/dedis/cothority/services/identity"
+	"gopkg.in/codegangsta/cli.v1"
+)
+
+// loadConfig will return nil if the config-file doesn't exist. It tries to
+// load the file given in configFile.
+func loadConfig() error {
+	log.Lvl2("Loading from", configFile)
+	buf, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil
+		}
+		return err
+	}
+	_, msg, err := network.UnmarshalRegistered(buf)
+	if err != nil {
+		return err
+	}
+	var ok bool
+	clientApp, ok = msg.(*identity.Identity)
```

Comment:
Same comments as above, here you read the config file into a global variable; it should probably be avoided if possible.
It also seems less *magical*:
```
func loadConfig() error  
```
 => Where does the config file comes from  and what does it read, where do I get the config ?
vs 
```
func loadConfig(config string) (Identity,error) 
```
The latter seems more explicit and less prone to *magic hazards* .
Do you think it's possible ? Again I'm not knowledgeable enough with the code but if possible that'd be really great.